### PR TITLE
docs(DIP29): random beacon for llmq selection

### PR DESCRIPTION
**Pull request by thephez**
This DIP proposes leveraging DVRFs (Decentralized [Verifiable Random Functions](https://en.wikipedia.org/wiki/Verifiable_random_function)) to create decentralized randomness inside the Dash ecosystem in a secure, verifiable manner.

Preview: https://github.com/dashpay/dips/blob/dip29-random-beacon/dip-0029.md

### CODE REVIEW

File: dip-0029.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+* Let `CB_CL(h)` be the ChainLock included in the coinbase of the block at height `h`.
+* Let `H_diff(h)` be the height difference included in the coinbase of the block at height `h`.
+
+Relevant block information is specified in the format:
+
+```cpp
+[h, CB_CL(h), H_diff(h)] = [h, CL(h -H_diff(h) -1), H_diff(h)].  
+```
+
+`[h, CB_CL(h), H_diff(h)`], `[h+1, CB_CL(h+1), H_diff(h+1)]` is a valid succession as long as
```
Comment (ogabrielides) [inline dip-0029.md:199]: `[h, CB_CL(h), H_diff(h)]`

**Review by ogabrielides — CHANGES_REQUESTED** [review]

**Review by ogabrielides — APPROVED** [review]
LGTM

**Review by UdjinM6 — APPROVED** [review]
LGTM

**Review by VirgileBa — APPROVED** [review]

**Review by PastaPastaPasta — APPROVED** [review]

**Review by PastaPastaPasta — APPROVED** [review]

**Review by ogabrielides — APPROVED** [review]
re-ACK

**Pull request merged**